draft-regext-rdap-x-media-type.md

```diff
@@ -189,11 +189,36 @@ Provisional Registration: No
 
 # Acknowledgements
 
-Pawel Kowalik provided ideas and feedbacks that have contributed to
+Pawel Kowalik and James Mitchel have provided ideas and feedbacks that have contributed to
 the content of this document.
 
 {backmatter}
 
+# Using the Vary Header {#vary_header}
```

Comment:
Should this section be right after the "Using The RDAP-X Media Type" section? Or, is it intended to be an appendix?

Reply:
I think it should be an appendix since it really isn't normative.

Reply:
OK